app/root.jsx

```diff
@@ -29,6 +29,7 @@ export default function App() {
         <meta charSet="utf-8" />
         <meta name="viewport" content="width=device-width,initial-scale=1" />
         <script defer data-domain="getalby.com" src="https://squirrel.getalby.com/js/plausible.js"></script>
+        <script>window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }</script>
```

Comment:
we can not use `window` here because of remix (rendering things on the server) 
I don't know yet how this would be done in the remix context